install.md

```diff
@@ -51,6 +55,101 @@ If there's a version or operating system that is missing, please [open an issue]
 
 For more information, please follow the instructions in the [CRI-O packaging repository.](https://github.com/cri-o/packaging/blob/main/README.md).
 
+## Install CRI-O on Flatcar with Sysexts
+
+Installing CRI-O on Flatcar Linux with support for systemd extensions (sysexts),
```

Comment:
```suggestion
Installing CRI-O on Flatcar Container Linux with support for systemd extensions (sysexts),
```

Reply:
We would use either "Flatcar" or "Flatcar Container Linux", if possible, and as appropriate.